### PR TITLE
Adding 2018-02-01 Traffic Manager API version.

### PR DESCRIPTION
**Pull request by hrkulkarMsft**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by hrkulkarMsft** [comment]
@fearthecowboy : This references this PR https://github.com/Azure/azure-rest-api-specs-pr/pull/395

**Comment by MikhailTryakhov** [comment]
@dsgouda  @hrkulkarMsft  unfortunately February branch contribution is hold many months ago. Unfortunately we do not accept such tricks here.
Also no need to push changes dierctly in master - please use Network-2018-05-01 branch and 2018-05-01 branch only to push changes.
Details and scenarios are able by [link](https://microsoft.sharepoint.com/teams/WAG/AzureNetworking/_layouts/OneNote.aspx?id=%2Fteams%2FWAG%2FAzureNetworking%2FOneNotes%2FNRP%20migrated&wd=target%28Hydra%20%2B%20SDK.one%7C96F7B95B-A378-4CA5-903B-C88F894873EF%2FSwagger%20-%3E%20SDK%20-%3E%20PowerShell%7C71B816ED-DAE5-4907-94D8-DE7EE13278A0%2F%29
onenote:https://microsoft.sharepoint.com/teams/WAG/AzureNetworking/OneNotes/NRP%20migrated/Hydra%20+%20SDK.one#Swagger%20-%3e%20SDK%20-%3e%20PowerShell&section-id={96F7B95B-A378-4CA5-903B-C88F894873EF}&page-id={71B816ED-DAE5-4907-94D8-DE7EE13278A0}&end)

**Comment by dsgouda** [comment]
Thanks @MikhailTryakhov 
This is a little confusing
@hrkulkarMsft does your RP have anything to do with `Microsoft.Network`? If yes, why is it under its own directory in the repo, if not, why is it named as `trafficmanager\Microsoft.Network`
We need to make sure the changes you are making here are valid.

File: specification/trafficmanager/resource-manager/readme.md [hunk 1]
```diff
@@ -37,6 +37,39 @@ input-file:
...
+- Microsoft.Network/preview/2017-09-01-preview/trafficmanageranalytics.json
+
+# Needed when there is more than one input file
+override-info:
+  title: TrafficManagerManagementClient
+
+directive:
+  - suppress: R3023
+    reason: it's implemented in the main network spec
+    approved-by: "@fearthecowboy"
```
Comment (dsgouda) [inline specification/trafficmanager/resource-manager/readme.md]: @fearthecowboy Please confirm your approval for these suppressions

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks like most of the spec has already been reviewed in the `2018-03-01` version
The CI failures must be resolved before approval

File: specification/trafficmanager/resource-manager/readme.md (hunk 1 above)
Comment (fearthecowboy) [reply specification/trafficmanager/resource-manager/readme.md]: Yes, these suppressions are ok. 

As far as I understand, they have things in the `Microsoft.Network` namespace, but it's still part of their RP.  (we've got a couple cases of this elsewhere)

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**